tests/DocumentTest.php

```diff
@@ -418,6 +461,252 @@ public function testFromFile()
+
+        self::assertInstanceOf('Sokil\Vast\Document', $document);
+
+        $adSections = $document->getAdSections();
+        /** @var InLine $adSection */
+        $inlineAd = $adSections[0];
+        $creatives = $inlineAd->getCreatives();
+
+        // test with empty VideoClicks tag
+        /** @var \Sokil\Vast\Creative\AbstractLinearCreative $creative */
+        $creative = $creatives[0];
+        self::assertEquals(
+            array(), $creative->getVideoClicksCustomClick()
+        );
+        self::assertEquals(
+            array(),
+            $creative->getVideoClicksCustomClick()
+        );
+        self::assertEquals(
+            '',
+            $creative->getVideoClicksClickThrough()
+        );
+
+        // test w/o VideoClicks tag/** @var InLine $adSection */
+        $inlineAd = $adSections[1];
+        $creatives = $inlineAd->getCreatives();
+        /** @var \Sokil\Vast\Creative\AbstractLinearCreative $creative */
+        $creative = $creatives[0];
+        self::assertEquals(
+            array(), $creative->getVideoClicksCustomClick()
+        );
+        self::assertEquals(
+            array(),
+            $creative->getVideoClicksCustomClick()
+        );
+        self::assertEquals(
+            '',
+            $creative->getVideoClicksClickThrough()
+        );
+    }
+
+    /**
+     * test tracking events retrieval
+     */
+    public function testTrackingEvents()
+    {
+        $factory = new Factory();
+        $document = $factory->fromFile(__DIR__ . '/data/vastTrackingEvents.xml');
+
+        self::assertInstanceOf('Sokil\Vast\Document', $document);
+
+        $adSections = $document->getAdSections();
+
+        // test with normal tracking events
+        /** @var InLine $adSection */
+        $inlineAd = $adSections[0];
+        $creatives = $inlineAd->getCreatives();
+        /** @var \Sokil\Vast\Creative\AbstractLinearCreative $creative */
+        $creative = $creatives[0];
+        self::assertSame(
+            array(
+                'start' => array('http://ad.server.com/trackingevent/start', ),
+                'pause' => array('http://ad.server.com/trackingevent/pause', ),
+            ),
+            $creative->getTrackingEvents()
+        );
+
+        // test with no tracking events
+        /** @var InLine $adSection */
+        $inlineAd = $adSections[1];
+        $creatives = $inlineAd->getCreatives();
+        /** @var \Sokil\Vast\Creative\AbstractLinearCreative $creative */
+        $creative = $creatives[0];
+        self::assertEquals(
+            array(),
+            $creative->getTrackingEvents()
+        );
+
+        // test with empty tracking events tag
+        /** @var InLine $adSection */
+        $inlineAd = $adSections[2];
+        $creatives = $inlineAd->getCreatives();
+        /** @var \Sokil\Vast\Creative\AbstractLinearCreative $creative */
+        $creative = $creatives[0];
+        self::assertEquals(
```

Comment:
this tests test to much in one method. If test will fail, it will be difficult to understand where problem is. test must test one thing.